integration/tests/posit/connect/test_users.py

```diff
@@ -1,29 +1,79 @@
 from posit import connect
 
 
-class TestAttributeContent:
+class TestUser:
+    @classmethod
+    def setup_class(cls):
+        cls.client = client = connect.Client()
+        cls.aron = client.users.create(
+            username="aron", email="aron@example.com", password="s3cur3p@ssword"
+        )
+        cls.bill = client.users.create(
+            username="bill", email="bill@example.com", password="s3cur3p@ssword"
+        )
+        cls.cole = client.users.create(
+            username="cole", email="cole@example.com", password="s3cur3p@ssword"
+        )
```

Comment:
For my own understanding: why are three users being tested? Are they different in some way?

Reply:
They probably aren't strictly necessary, but I wanted to ensure the filtering worked correctly.